test/integration/targets/aws_guardduty_detector/tasks/main.yml

```diff
@@ -0,0 +1,48 @@
+---
+
+- name: set connection information for all tasks
+  set_fact:
+    aws_connection_info: &aws_connection_info
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      #security_token: "{{ security_token }}"
```

Comment:
This needs to be uncommented to run in CI. You can make it compatible with non-STS credentials by using "{{ security_token | omit() }}" or "{{ security_token | default() }}"